UI/SidePanel.cpp

```diff
@@ -2531,6 +2539,14 @@ void SidePanel::PlanetPanelContainer::RefreshAllPlanetPanels() {
         (*it)->Refresh();
 }
 
+void SidePanel::PlanetPanelContainer::ShowScrollbar(){
+	DoPanelsLayout();
+}
+
+void SidePanel::PlanetPanelContainer::HideScrollbar(){
```

Comment:
If this did have a { at the end of a line, it should have a space between ) and {